test/integration/targets/openssl_pkcs12/aliases

```diff
@@ -0,0 +1 @@
+posix/ci/group1
```

Comment:
`destructive` and `needs/root` must be added because `setup_openssl` use `package` module.